tagmanagementdispatcher/src/main/java/com/tealium/tagmanagementdispatcher/WebViewLoader.kt

```diff
@@ -270,19 +272,22 @@ class WebViewLoader(
         return queryParams.toMap()
     }
 
-    private fun initializeWebView() {
-        scope.launch {
-            if (hasReachedMaxErrors()) return@launch
+    internal fun initializeWebView() : Deferred<Unit> {
```

Comment:
More a curious question: Is the Deferred return type just for tests really? I assume we trigger the method and forget about the result at the call sites.